src/som/interpreter/ReturnException.java

```diff
@@ -0,0 +1,26 @@
+package som.interpreter;
+
+import som.vmobjects.SAbstractObject;
+
+public class ReturnException extends RuntimeException {
```

Comment:
You may already want to use Truffle's `ControlFlowException` class as superclass here.